mmv1/templates/validator/resource_conversion.go.erb

```diff
@@ -22,7 +22,8 @@ import (
     resource_name = product_ns + object.name
     properties = object.all_user_properties
     # See discussion on asset name here: https://github.com/GoogleCloudPlatform/magic-modules/pull/1520
-    asset_name_template = '//' + product_ns.downcase + '.googleapis.com/' + (!object.self_link.nil? && !object.self_link.empty? ? object.self_link : object.base_url + '/{{name}}')
+		product_backend_name=@base_url.split('://')[1].split('.googleapis.com')[0]
```

Comment:
nit: looks like the indentation here is incorrect

Reply:
updates